metabase/metabase
# fix line charts broken due to stacking settings

Closes https://github.com/metabase/metabase/issues/40989

### Description

Line charts got broken by https://github.com/metabase/metabase/pull/37769 because adding `"stackable.stack_type"` read dependencies to tooltip and legend settings causes an unexpected behavior with the existing settings implementation. 

### How to verify

Create a stacked bar chart and then switch the display type to `line` and ensure it works.
